lib/plugin.js

```diff
@@ -73,8 +73,16 @@ function startPageTracking(ctx) {
 }
 
 export default function (ctx, inject) {
-  const initialized = {<%= (options.autoInit && options.id) ? ` '${options.id}': true ` : '' %>}
-  ctx.$gtm = process.client ? gtmClient(ctx, initialized) : gtmServer(ctx, initialized)
+  const runtimeConfig = (ctx.$config && ctx.$config.gtm) || {}
+  const autoInit = <%= options.autoInit %>
```

Comment:
both `id` and `autoInit` _may_ be also inferred from runtimeConfig. What do you think using `{ _id, autoInit, ...runtimeConfig}`?

Reply:
This changes are no longer needed, since the module itself will be runtime, we can grab the configurations before loading the plugin.

Reply:
_I'm reopening this topic, since we back to plugin again_ 🙈

As discussed in the other thread, the `id` set thought module options is appended by the module itself. Instead of overwriting the default behavior, I included the runtime id as an additional script.

Is this ok?